### ISSUE TITLE
.withContentContains => Unexpected MimeMessage content 	at net.kemitix.wiser.assertions.WiserAssertions.getMimeMessageBody(WiserAssertions.java:190)

### ISSUE THREAD

**Issue by bodtx** [kemitix/wiser-assertions#6]
Hello when I create a mail with spring mail:
```
SimpleMailMessage message = new SimpleMailMessage();
....
new JavaMailSenderImpl().send(message);
```

I check the mail in my Junit and all is going weel except the withContentContains:

```
assertReceivedMessage(wiser).from("test@...").to("...")
				.withSubject("...")
				.withContentContains(...);
```

which throws Unexpected MimeMessage content
	at net.kemitix.wiser.assertions.WiserAssertions.getMimeMessageBody(WiserAssertions.java:190)

**Comment by kemitix** [comment]
Thanks for reporting this. It is fixed in 0.3.1, which I've released to the maven repositories. It should be available shortly, once it has been indexed.

**Comment by bodtx** [comment]
Great, it works fine with 0.3.1, thx for theese assertions.
